fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -700,14 +700,8 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType,
                         if (ModelSupport.isResourceType(resType)) {
                             resourceTypes.add(resType);
                         } else {
-                            String msg = "_type search parameter has invalid resource type:" + resType;
-                            if (lenient) {
-                                // TODO add this to the list of supplemental warnings?
```

Comment:
I noticed we lost a couple TODOs in the scope of these changes.  We should probably talk about these and possibly open an enhancement request so that we don't lose this thought.
The idea is to include a OperationOutcome with an Issue of type warning with information about the ignored parameter.  However, I think that clients would only receive that OperationOutcome if either:
A. they specify a "return preference" of OperationOutcome (via the Prefer header); or
B. something else goes wrong causing us to return an OperationOutcome

For search, we would need to set the Bundle.entry.search.mode value of this OperationOutcome entry to `outcome`.

Reply:
I think this makes sense. I assume it would apply to all instances where we're in lenient mode and we ignore a parameter. Are there other cases where we ignore parameters?